### PR TITLE
Adityakumar.Pycalctrice version 1.0.0

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/24020)

### CODE REVIEW

File: manifests/a/Adityakumar/Pycalctrice/1.0.0/Adityakumar.Pycalctrice.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+LicenseUrl: https://github.com/Aditya5k/Pycalctrice/blob/main/LICENSE
+Copyright: Aditya kumar
+CopyrightUrl: https://github.com/Aditya5k/Pycalctrice/blob/main/LICENSE
+ShortDescription: Pycalctrice Setup
+Description: https://github.com/Aditya5k/Pycalctrice/blob/main/README.md
+Tags:
+- Python
+- cli
+- calculator
+- x64
```
Comment (vedantmgoyal2009) [inline manifests/a/Adityakumar/Pycalctrice/1.0.0/Adityakumar.Pycalctrice.locale.en-US.yaml:24]:
```suggestion
PublisherUrl: https://github.com/Aditya5k
PublisherSupportUrl: https://github.com/Aditya5k/Pycalctrice/issues
#PrivacyUrl: 
Author: Aditya kumar
PackageName: Pycalctrice
PackageUrl: https://github.com/Aditya5k/Pycalctrice
License: MIT License
LicenseUrl: https://raw.githubusercontent.com/Aditya5k/Pycalctrice/main/LICENSE
Copyright: Copyright (c) 2021 Aditya kumar
CopyrightUrl: https://raw.githubusercontent.com/Aditya5k/Pycalctrice/main/LICENSE
ShortDescription: A simple CLI calculator in Python.
Tags:
- python
- cli
- calculator
```

**Review by vedantmgoyal9 — CHANGES_REQUESTED** [review]

**Review by vedantmgoyal9 — COMMENTED** [review]
Sorry, but I think it will not get passed due to Windows SmartScreen.
image

**Comment by Aditya5k** [comment]
any solutions

**Comment by vedantmgoyal2009** [comment]
While making your installer, fill in complete information and make sure it is signed.

**Comment by Aditya5k** [comment]
ok

**Comment by Aditya5k** [comment]
it got passed 😀

**Comment by vedantmgoyal2009** [comment]
This is the first time I have seen it, on the local machine it triggers SmartScreen Defender but here, it gets passed.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**